src/zerocoin.cpp

```diff
@@ -53,26 +53,28 @@ static bool CheckZerocoinSpendSerial(CValidationState &state, CZerocoinTxInfo *z
 }
 
 bool CheckSpendZcoinTransaction(const CTransaction &tx,
-                                libzerocoin::CoinDenomination targetDenomination,
+                                vector<libzerocoin::CoinDenomination> targetDenominations,
```

Comment:
I think it will be better if we change to `const vector<libzerocoin::CoinDenomination>& targetDenominations` instead to prevent copying the whole vector.